samples/bootloader/README.rst

```diff
@@ -185,9 +185,10 @@ Building and running
 .. include:: /includes/build_and_run.txt
 
 .. caution::
-   The NSIB should be included as a child image in a multi-image build, rather than being built stand-alone.
+   The NSIB should be included as a child image in a :ref:`multi-image build <ug_multi_image>`, rather than being built stand-alone.
    While it is technically possible to build the NSIB by itself and merge it into other application images, this process is not supported.
    To reduce the development time and potential issues with this route, let the existing |NCS| infrastructure for multi-image builds handle the integration.
+   See :ref:`ug_multi_image_defining` for more information.
```

Comment:
This documentation is intended for people who define their own child images. Users of NSIB will not be adding it as a multi image, but instead NSIB will automatically be included if they set CONFIG_SECURE_BOOT. I suggest that we do not mention this link, but instead say. "NSIB will automatically be added as a child image if CONFIG_SECURE_BOOT is set in an application.". I think that is the information users will need in practice.